test/integration/lib/test-actions/index.ts

```diff
@@ -119,8 +119,11 @@ async function _startAndAcceptInteraction(
   assert(interactId)
 
   // Accept
+  const url = new URL(senderWalletAddress.authServer)
+  url.port = '3109'
```

Comment:
I think this port would depend on the ASE itself, right? It's either 3109 for cloud-nine-wallet or 4109 for happy-life-bank

Reply:
I think since you have added the `interactionServer` in the env file/config, you can just pass it through, pass `deps: TestActionDeps` into the function and use  `deps.sendingASE.config.interactionServer` directly